flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -66,14 +66,13 @@
 import org.apache.flink.runtime.state.internal.InternalReducingState;
 import org.apache.flink.runtime.state.internal.InternalValueState;
 import org.apache.flink.runtime.util.BlockerCheckpointStreamFactory;
+import org.apache.flink.shaded.guava18.com.google.common.base.Joiner;
```

Comment:
I don't think this will pass check style checks, if not, we still don't allow this usually. (I know that the import was already there previously 😅 )

Reply:
IIRC, checkstyle actually _forced_ me to position the import there.